compiler/msgs.nim

```diff
@@ -9,7 +9,7 @@
 
 import
   options, strutils, os, tables, ropes, terminal, macros,
-  lineinfos, pathutils
+  lineinfos, pathutils, strformat
```

Comment:
`nim --colors -d:optUseColors --eval:'var a: int = "abc"'`

![image](https://user-images.githubusercontent.com/2194784/102338511-ca59a580-3f48-11eb-8c60-906c89f0b8b0.png)

"abc"
shouldn't be in double quotes, since it's in color already just abc is fine

Reply:
my bad, the double quotes comes from string here, works as intented